packages/button/src/css/index.ts

```diff
@@ -83,25 +81,25 @@ export default {
   [`.psds-button--size-${vars.sizes.xSmall}`]: {
     fontSize: type.fontSize100,
     letterSpacing: type.letterSpacingLooser,
-    padding: `0 ${layout.spacingXSmall}`,
+    padding: `4px 8px`,
```

Comment:
Thinking we want to use layout spacing values where possible. I can help you with that if you need it.

Reply:
@EdwardIrby I specifically removed them inside the component because there aren't spacing variables for all of the values I needed. So in some cases I'd end up with a mix like this:

`padding: \`6px ${layout.spacingSmall}\``

I wasn't really sure what value the variable was providing. Using the actual pixel values seemed clearer to me. I'm not sure about you, but I have to look up the pixel values for each spacing variable whenever I use them.